config/webpack.config.js

```diff
@@ -231,7 +231,8 @@ module.exports = env => {
       host: devServer.host,
       port: devServer.port,
       headers: { 'Access-Control-Allow-Origin': '*' },
-      hot: true
+      hot: true,
+      disableHostCheck: true,
```

Comment:
We can override the same option here without requiring a forked version of the dev server
```suggestion
      disableHostCheck: true,
      https: { spdy: { protocols: ['http/1.1'] }},
```

Reply:
I tried exactley to psss the same option, though it didn't work :(

Reply:
I suspect passing `--https` as an argument may override the option here.

Reply:
@tbrisker if I understand JS hash decomposition here: https://github.com/webpack/webpack-dev-server/blob/bc63a7191d947e030056c2c604743692c297b1b7/lib/Server.js#L537 the protocols node would be overridden, no matter what comes in options.https.

Reply:
@ShimShtein isn't that overriden only when `options.http2` is set? I'm guessing that @ekohl is right and that passing a command line argument might override the configuration

Reply:
I added a `console.log(options)` inside the dev server code right before the line @laviro changed and i did get my options there when running locally. when i add the `--https` indeed that option gets overriden though.

Reply:
Note that we are currently using version 2.11.5 of the webpack dev server, i wonder if we could upgrade the dev server without upgrading webpack itself?

Reply:
and here is where it gets overridden when `--https` is passed:
https://github.com/webpack/webpack-dev-server/blob/v2/lib/Server.js#L399-L410

Reply:
I wonder if we could upgrade webpack :thinking:  xD

Reply:
@MariaAga how's that going? :smile:

Reply:
@jeremylenz Slowly :laughing: